string direction
int64 angle
---
uint8 turning_status
